PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterCharacterScripts/Sound.rbxmx

```diff
@@ -0,0 +1,381 @@
+<roblox xmlns:xmime="http://www.w3.org/2005/05/xmlmime" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.roblox.com/roblox.xsd" version="4">
+	<External>null</External>
+	<External>nil</External>
+	<Item class="Script" referent="RBXc83eeca86f73463493b2478b693fc06d">
+		<Properties>
+			<bool name="Disabled">false</bool>
+			<Content name="LinkedSource"><null></null></Content>
+			<string name="Name">Sound</string>
+			<string name="ScriptGuid"></string>
+			<ProtectedString name="Source"><![CDATA[--[[
+	This script creates sounds which are placed under the character head.
+	These sounds are used by the "LocalSound" script.
+]]--
+--humanoidSoundNewLocal.rbxmx--
+
+function CreateNewSound(name, id, looped, pitch)
+	local sound = Instance.new("Sound")
+	sound.SoundId = id
+	sound.Name = name
+	sound.archivable = false
+	sound.Parent = script.Parent.Head
```

Comment:
is this script created on the server? I it seems be of Script type, not LocalScript. Is this to allow FE games to hear the sounds of humanoids? Also, should we verify Head exists?